clients/windows/src/input/file_drop.rs

```diff
@@ -71,12 +70,14 @@ pub fn handle(app: &mut WgpuLockbook, object: Option<IDataObject>) -> bool {
                 break;
             }
 
+            // todo: support additional formats (including custome/registerd formats)
             let format = CLIPBOARD_FORMAT(rgelt[0].cfFormat);
-            let mut format_name = [0u16; 1000];
             let is_predefined_format = format_str(format).is_some();
-            let is_registered_format =
-                unsafe { GetClipboardFormatNameW(format.0 as _, &mut format_name) != 0 };
-            if !is_predefined_format && !is_registered_format {
+            // use windows::Win32::System::DataExchange::*,
+            // let mut format_name = [0u16; 1000];
+            // let is_registered_format =
+            // unsafe { GetClipboardFormatNameW(format.0 as _, &mut format_name) != 0 };
+            if !is_predefined_format {
```

Comment:
app was getting data for an unnecessary format (probably the file data itself instead of the path) and it was taking a long time, causing the app to hang when you drop a file because it's importing the file itself (in addition to the path) on the UI thread